src/permission/rules-map.js

```diff
@@ -60,6 +61,14 @@ RULES_MAP[ C.TOPIC.RPC ].actions = {};
 RULES_MAP[ C.TOPIC.RPC ].actions[ C.ACTIONS.SUBSCRIBE ] = RULE_TYPES.PROVIDE;
 RULES_MAP[ C.TOPIC.RPC ].actions[ C.ACTIONS.REQUEST ] = RULE_TYPES.REQUEST;
 
+RULES_MAP[ C.TOPIC.PRESENCE ] = {};
+RULES_MAP[ C.TOPIC.PRESENCE ].section = 'presence';
+RULES_MAP[ C.TOPIC.PRESENCE ].actions = {};
+RULES_MAP[ C.TOPIC.PRESENCE ].actions[ C.ACTIONS.SUBSCRIBE ] = RULE_TYPES.SUBSCRIBE;
+RULES_MAP[ C.TOPIC.PRESENCE ].actions[ C.ACTIONS.SUBSCRIBE ] = RULE_TYPES.SUBSCRIBE;
```

Comment:
At the moment, we subscribe to both login and logout events. How to fine tune permissions to allow subscriptions to just login or just logout events - or do we even need to worry about that?